api/Blob.json

```
@@ -347,7 +347,7 @@
               "version_added": false
             },
             "opera": {
-              "version_added": false
+              "version_added": "75"
```

```suggestion
              "version_added": "63"
```

I tested https://mdn-bcd-collector.appspot.com/tests/api/Blob/text in Opera 62 and 63 on BrowserStack to confirm this. This is what you get from `npm run mirror -- -m always opera api.Blob.text`, since Opera 63 is baed on Blink 76.